src/components/__tests__/page-details.test.jsx

```diff
@@ -10,13 +10,16 @@ describe('page-details', () => {
   simplePage.versions.forEach(version => {
     version.capture_time = new Date(version.capture_time);
   });
-  const match = { params: { pageId: simplePage.uuid }};
-  const createMockApi = () => {
-    return Object.assign(Object.create(WebMonitoringDb.prototype), {
+  const match = {params: {pageId: simplePage.uuid}};
+
+  const createMockApi = overrides => Object.assign(
+    Object.create(WebMonitoringDb.prototype),
+    {
       getPage: jest.fn().mockResolvedValue(simplePage),
-      getVersions: jest.fn().mockResolvedValue(simplePage.versions)
-    });
-  };
+      getVersions: jest.fn().mockResolvedValue(simplePage.versions),
+    },
+    overrides
+  );
```

Comment:
If you’re going to reformat this to a different syntax, I’d rather see:

```js
function createMockApi (overrides) {
  // ...
}
```

…than an arrow expression being assigned to a variable. I think it’s *dramatically* clearer and easier to read for this kind of thing.

Reply:
done!